.github/workflows/test.yml

```diff
@@ -331,3 +331,234 @@ jobs:
             
             echo "::endgroup::"
           }
+
+  
+  
+  correct_bindgen:
```

Comment:
Maybe pick a name that is a bit more informative? If the main point is no cross compile call it `bindgen_no_cross_compile` maybe?

Also, could you add a few comments explaining what this test is meant to do compared to the other tests?

Reply:
@clauswilke,
I added some comments. Currently, we have 'old' bindgen tests and new 'no cross-compilation' bindgen tests, which do pretty much the same thing. Should we remove 'old' bindgen job because it does not work for Windows (at all)?

Reply:
> Should we remove 'old' bindgen job because it does not work for Windows (at all)?

Yes, I think that's fine.

Reply:
Ok, removed old tests.